_studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_hw.cpp

```diff
@@ -794,7 +817,7 @@ mfxStatus VideoDECODEVP8_HW::DecodeFrameCheck(mfxBitstream *p_bs, mfxFrameSurfac
         m_p_video_accelerator->EndFrame();
     }
 
-    sts = GetOutputSurface(pp_surface_out, p_surface_work, memId);
+    sts = FillOuputSurface(pp_surface_out, p_surface_work, memId);
```

Comment:
I suggest to left GetOutputSurface as is, not mask the call within FillOutputSurface.